build-system/tasks/runtime-test.js

```diff
@@ -42,6 +42,17 @@ const preTestTasks = argv.nobuild ? [] : (
 const extensionsCssMapPath = 'EXTENSIONS_CSS_MAP';
 const batchSize = 4; // Number of Sauce Lab browsers
 
+const chromeBase = argv.chrome_canary ? 'ChromeCanary' : 'Chrome';
+
+const formattedFlagList = [];
+if (argv.chrome_flags) {
+  const flagList = argv.chrome_flags.split(',');
+  flagList.forEach(flag => {
+    formattedFlagList.push('--'.concat(flag));
+  });
+}
```

Comment:
It's sufficient for `formattedFlagList` to be a top-level `const`. I think it'll be cleaner to move the actual processing of flags back to `getConfig()`. Sorry about the extra work!